lib/mix/tasks/docs.ex

```diff
@@ -249,19 +249,23 @@ defmodule Mix.Tasks.Docs do
   defp normalize_source_beam(options, config) do
     compile_path =
       if Mix.Project.umbrella?(config) do
-        umbrella_compile_paths()
+        umbrella_compile_paths(get_ignored_apps(options))
```

Comment:
`get_ignored_apps(options)`  could be written as `Keyword.get(options, :ignore_apps, [])`.